code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1509,6 +1510,7 @@
 	victim.dead_ticks = 0
 	ADD_TRAIT(victim, TRAIT_STASIS, TRAIT_STASIS)
 	X.eject_victim(TRUE, starting_turf)
+	X.biomass = min(X.biomass + 15, 100)
```

Comment:
Add a xeno proc that has a clamp that handles this and call that here, so you don't have to limit it each time